firebase.json

```diff
@@ -0,0 +1,64 @@
+{
+  "firestore": {
+    "rules": "firestore.rules",
+    "indexes": "firestore.indexes.json"
```

Comment:
Can we moves these files out of the root of the repo? If so that'd be great to keep it clean!

Reply:
It may be possible with some config settings. I’ve usually left them, but I do see your point.